setup.py

```diff
@@ -23,6 +23,7 @@
         "onnxruntime>=1.10.0",
         "paramiko==2.11.0",
         "torch>=1.12.1",
+        "torch_geometric>=2.3.0",
```

Comment:
`torch_geometric` is known for seemingly installing correctly and then causing a segmentation fault when imported. It would be great to test for that.

This should be fairly simple. Modify the workflows file `.github/workflows/test_mlagility.yml ` the following way.

Replace:
```
- name: Check installation of requirements.txt and their compatibility with mlagility
        shell: bash -el {0}
        run: |
          bash src/mlagility/cli/setup_venv.sh
          conda activate tracker_slurm
          python -m pip check
          conda deactivate
```
by

```          
- name: Check installation of requirements.txt and their compatibility with mlagility
        shell: bash -el {0}
        run: |
          bash src/mlagility/cli/setup_venv.sh
          conda activate tracker_slurm
          python -m pip check
          python -c "import torch_geometric"
          conda deactivate
```